core/tauri-utils/src/config.rs

```diff
@@ -919,9 +919,18 @@ pub struct WindowConfig {
   /// The max window height.
   #[serde(alias = "max-height")]
   pub max_height: Option<f64>,
-  /// Whether the window is resizable or not.
+  /// Whether the window is resizable or not. When resizable is set to false, native window's maximize button is automatically disabled.
   #[serde(default = "default_resizable")]
   pub resizable: bool,
+  /// Whether the window's native maximize button is enabled or not. If resizable is set to false, this setting is ignored. Note that this setting is not supported on Linux, iOS and Android.
```

Comment:
you also need to add the platform-specific notes here as well

Reply:
Should the docs in this file have the same style as docs in the rest of the PR? I wrote them out differently, because I saw that people wrote platform-specific comments at the end of the line as a "note" in this file.

Examples: [1](https://github.com/tauri-apps/tauri/blob/dfb5f52d3cbe44ce4607cd875b9632093b9123b1/core/tauri-utils/src/config.rs#L955) ;  [2](https://github.com/tauri-apps/tauri/blob/dfb5f52d3cbe44ce4607cd875b9632093b9123b1/core/tauri-utils/src/config.rs#L958) ; [3](https://github.com/tauri-apps/tauri/blob/dfb5f52d3cbe44ce4607cd875b9632093b9123b1/core/tauri-utils/src/config.rs#L963)

Do you recon I should rewrite in the style [A](https://github.com/tauri-apps/tauri/blob/dfb5f52d3cbe44ce4607cd875b9632093b9123b1/core/tauri-utils/src/config.rs#L936) or [B](https://github.com/tauri-apps/tauri/pull/6665/files#diff-a188a5964932c9b690a134501222d34882286e8d69a3f70534d09108822ddab4R666)?

Reply:
lets use this style https://github.com/tauri-apps/tauri/pull/6665/files#diff-a188a5964932c9b690a134501222d34882286e8d69a3f70534d09108822ddab4R681-R685 since it there is more info that need to be explained other than just what platform is supported.

Reply:
[Fixed](https://github.com/tauri-apps/tauri/pull/6665/commits/e4e6bda03dd6444bf118641dd41c23e4f549638d)